build.gradle

```diff
@@ -85,16 +85,31 @@ springBoot {
 
 rat {
     excludes = [
-            'node_modules/**',
+            'client/node_modules/**',
+            'client/dist/**',
+            'client/.babelrc',
+            'client/.editorconfig',
+            'client/.eslintignore',
+            'client/.eslintrc.js',
+            'client/.postcssrc.js',
+            'client/README.md',
+            'client/config/',
+            'client/static/.gitkeep',
+            'client/test/unit/.eslintrc',
+            'client/test/unit/coverage/',
+            'client/test/e2e/custom-assertions/elementCount.js',
+            'client/test/e2e/nightwatch.conf.js',
+            'client/test/e2e/runner.js',
+            'client/test/e2e/specs/test.js',
```

Comment:
We can probably make this a little more succinct, but I'll do a review of the rat config before the next holdmail release.